pkg/api/service/annotations.go

```diff
@@ -25,4 +33,54 @@ const (
 	//
 	// Not all cloud providers support this annotation, though AWS & GCE do.
 	AnnotationLoadBalancerSourceRangesKey = "service.beta.kubernetes.io/load-balancer-source-ranges"
+
+	// An annotation that denotes if this Service desires to route external traffic to local
+	// endpoints only. This preserves Source IP and avoids a second hop.
+	AnnotationExternalTraffic            = "service.alpha.kubernetes.io/external-traffic"
```

Comment:
Ah, I think that this replaces `service.alpha.kubernetes.io/node-local-loadbalancer` in the proposal?  Please keep the document and the implementation in sync, it makes it easier for other implementors.